app/src/main/res/values/strings.xml

```diff
@@ -205,7 +205,7 @@
     <string name="clear_cookie_title">Clear reCAPTCHA cookies</string>
     <string name="recaptcha_cookies_cleared">reCAPTCHA cookies have been cleared</string>
     <string name="import_data_summary">Overrides your current history and subscriptions</string>
```

Comment:
I also wanna change, but I'm not sure what to change it to.

Reply:
I don't think that's needed: while importing a popup comes up asking whether to import settings or not

Reply:
Playlists are imported too, but this is not listed in the summary. What should we do about that?

Reply:
Oh, that should be noted though

Reply:
There might also be other stuff we missed too. Hmm...